### PR TITLE
[FAB-18046] IssuerRevocationPublicKey -> RevocationPublicKey

### PR DESCRIPTION
**Pull request by davidkhala**
IssuerRevocationPublicKey -> RevocationPublicKey

Update related idemix document

Signed-off-by: david <david-khala@hotmail.com>
Change-Id: Id9273e1243fa42194a8e7deb6e2ae5d5ea6abbff

#### Type of change
- Documentation update

#### Description

IssuerRevocationPublicKey -> RevocationPublicKey

Only the later is recognizable by configtxgen

#### Related issues

FAB-18046

### CODE REVIEW

**Comment by lindluni** [comment]
Thanks David,

Looks like there are two additional references on line `docs/source/idemix.rst:79` and `docs/source/idemix.rst:88`

**Comment by davidkhala** [comment]
@btl5037  Dear bret, it is sad that they are left there as intended. Because the original file in fabric-ca is indeed named as IssuerRevocationPublicKey. Developer have to use this file content but only after renaming its file name to fit `configtxgen`

Basically this issue come from fabric-ca-server miss-aligned with `configtxgen`

**Review by denali49 — APPROVED** [review]
lgtm, thank you for your contribution!  Approval for DOC REVIEW only.

File: docs/source/test_network.md [hunk 1]
```diff
@@ -501,7 +501,7 @@ The command will reveal the MSP folder structure and configuration file:
 organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/
 └── msp
     ├── IssuerPublicKey
-    ├── IssuerRevocationPublicKey
+    ├── RevocationPublicKey
```
Comment (lindluni) [inline docs/source/test_network.md:504]: The right thing to do here might be to change idemix itself to match the CA. I opened #1557 lets see what Ale, Artem, or Yacov say on this.

**Review by lindluni — CHANGES_REQUESTED** [review]
Blocking merge until we get a response from Ale, Artem or Yacov

File: docs/source/test_network.md (hunk 1 above)
Comment (davidkhala) [reply docs/source/test_network.md:504]: Great, instead of maintaining two names as confusion handler, Code change could be better!

**Comment by denyeart** [comment]
@adecaro Could you take a look?

**Comment by lindluni** [comment]
> @adecaro Could you take a look?

Can you actually take a look at this #1557, I think that would superseded this and we could close this one

**Comment by ale-linux** [comment]
#1557 was closed - should we close this one as well @btl5037?

**Comment by davidkhala** [comment]
@ale-linux The close action is Bret's mistake.

**Comment by lindluni** [comment]
It was a mistake. I was cleaning up branches in my Fabric repo and accidentally deleted the branch.

**Comment by lindluni** [comment]
Going to fix #1557 and reopen and address there

**Pull request closed without merging**